### ISSUE TITLE
Show calendar-loading spinner only after 3 seconds passed

### ISSUE THREAD

**Issue by georgehrke** [nextcloud/calendar#15]

**Comment by georgehrke** [comment]
There have been changes with regard to event rendering in full calendar 3.1.
Let's postpone this to 1.6, so we don't have to implement it twice.

**Comment by alexanderdd** [comment]
Why? I think it makes sense to show the spinner immediately when I activate the calendar, until it is loaded, so the state of the application ("loading") is clear?

**Comment by alexanderdd** [comment]
Can someone please explain why this is a good idea? I think it's a bad idea, see my comment above.

**Comment by ChristophWurst** [comment]
I assume if the spinner is just shown for like 1s then this is more distracting than anything. Aka flickering.

**Comment by tcitworld** [comment]
This is also pre vue-rewrite so it might be completely outdated.

**Comment by st3iny** [comment]
There is a spinner in the vue rewrite.
